docs/modules/ROOT/pages/access-control.adoc

```diff
@@ -202,3 +134,143 @@ TIP: A recommended configuration is to grant both roles to a secure governance c
+
+The AccessManager is designed around the concept of role:
+
+* Roles are granted to accounts (addresses) following a many-to-many approach. This means that each user can have one or multiple roles and multiple users can have the same role.
+* Access to a restricted function is restricted to one role. A function is defined by one https://docs.soliditylang.org/en/v0.8.20/abi-spec.html#function-selector[function selector] on one contract (called target). 
+
+For a call to be authorized, the caller must bear the role that is assigned to the current function (contract address + function selector).
+
+image::access-manager-functions.svg[Access Manager functions]
+
+=== Using `AccessManager`
+
+OpenZeppelin Contracts provides xref:api:access.adoc#AccessManager[`AccessManager`]. A contract that can be deployed and used out of the box. It sets an initial admin in the constructor who will be allowed to perform management operations.
+
+In order to restrict access to some functions of your contract, you should inherit from the xref:api:access.adoc#AccessManaged[`AccessManaged`] contract provided along with the manager so that the `restricted` modifier is available to protect any externally facing function. Note you will have to set the address of the Access Manager instance (xref:api:access.adoc#AccessManaged-authority--[authority]) in the constructor so the `restricted` modifier can check permissions by calling the correct manager.
+
+Here's a simple example of an xref:tokens.adoc#ERC20[`ERC20`] token that defines a `mint` functionality that is restricted by an xref:api:access.adoc#AccessManager[`AccessManager`]:
+
+```solidity
+include::api:example$access-control/AccessManagedERC20MintBase.sol[]
+```
+
+NOTE: Make sure you fully understand how xref:api:access.adoc#AccessManager[`AccessManager`] works before using it or copy-pasting the examples from this guide.
+
+Once the managed contract has been deployed, it is now under the manager's control, which will get the minter role assigned to an address and also allow the role to call the `mint` function.
+
+```javascript
+const MINTER = 42n; // Roles are uint64 (0 is reserved for the ADMIN_ROLE)
+
+// Allow the minter role to call the function selector
+// corresponding to the mint function
+await manager.setTargetFunctionRole(
+    myToken.address,
+    ['0x40c10f19'], // bytes4(keccak256('mint(address,uint256)'))
+    MINTER,
+    { from: initialAdmin }
+);
+```
+
+Even though each role has its own list of function permissions, each role member (`address`) will have an execution delay that will dictate how long the account should wait to execute a function that requires its role. Delayed operations must be xref:api:access.adoc#AccessManager-schedule-address-bytes-uint48-[`schedule`d] first on the Access Manager before they can be executed, either directly or using the Access Manager's xref:api:access.adoc#AccessManager-execute-address-bytes-[`execute`] function.
```

Comment:
Formatting is off, probably because of schedule'd. Maybe change to something like:
```suggestion
Even though each role has its own list of function permissions, each role member (`address`) will have an execution delay that will dictate how long the account should wait to execute a function that requires its role. Delayed operations must have the xref:api:access.adoc#AccessManager-schedule-address-bytes-uint48-[`schedule`] function called on them first in the Access Manager before they can be executed. This `schedule` call can occur either directly or using the Access Manager's xref:api:access.adoc#AccessManager-execute-address-bytes-[`execute`] function.
```

Reply:
> either directly or using the Access Manager's xref:api:access.adoc#AccessManager-execute-address-bytes-[`execute`] function.

It's not clear what "directly" is referring to.  Is it referring to the `schedule` call?

Reply:
Changed for "calling the target function" instead.